openedx/core/djangoapps/user_api/management/commands/email_opt_in_list.py

```diff
@@ -237,7 +237,7 @@ def _write_email_opt_in_prefs(self, file_handle, org_aliases, courses):
             email, full_name, course_id, is_opted_in, pref_set_datetime = row
             writer.writerow({
                 "email": email.encode('utf-8'),
-                "full_name": full_name.encode('utf-8'),
+                "full_name": full_name.encode('utf-8') if full_name else '',
                 "course_id": course_id.encode('utf-8'),
```

Comment:
@Shrhawk I didn't use the method call here because its just a `oneliner` and only being used at a single place, so I don't think method call is necessary. 
Thoughts?